ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/CaseLink.java

```diff
@@ -22,7 +22,7 @@ public class CaseLink {
   private List<ListValue<LinkReason>> reasonForLink;
 
   @JsonProperty("CreatedDateTime")
-  private LocalDate createdDateTime;
+  private LocalDateTime createdDateTime;
```

Comment:
@sjamihmcts - Is it better to add the date time format here? (remember we had some issues without the formatter)

Reply:
ok got you Rajaram. Will add the JsonFormat as well.

Reply:
yeah that's a good shout

Reply:
Added the JsonFormat attribute.